test/e2e/placement_apply_strategy_test.go

```diff
@@ -34,12 +33,9 @@ var _ = Describe("validating CRP when resources exists", Ordered, func() {
 	AfterAll(func() {
 		By("deleting created work resources on hub cluster")
 		cleanupWorkResources()
-
-		By("deleting created work resources on member cluster")
-		cleanWorkResourcesOnCluster(allMemberClusters[0])
 	})
 
-	Context("Test a CRP place objects successfully (client-side-apply)", Ordered, func() {
+	Context("Test a CRP place objects successfully (client-side-apply and allow co-own)", Ordered, func() {
 		BeforeAll(func() {
 			ns := appNamespace()
 			ns.SetOwnerReferences([]metav1.OwnerReference{
```

Comment:
is the  owner reference copied to the member cluster?

Reply:
here it creates the ns directly on the member cluster. are you suggesting that we create a ns with ownerreference in the hub to see if the ownerrefernces are copied to the member?